tutorials/assets_pipeline/importing_3d_scenes/available_formats.rst

```diff
@@ -0,0 +1,167 @@
+.. _doc_importing_3d_scenes_available_formats:
+
+Available 3D formats
+====================
+
+When dealing with 3D assets, Godot has a flexible and configurable importer.
+
+Godot works with *scenes*. This means that the entire scene being worked on in
+your favorite 3D modeling software will be transferred as close as possible.
+
+Godot supports the following 3D *scene file formats*:
+
+- glTF 2.0 **(recommended)**. Godot has full support for both text (``.gltf``)
+  and binary (``.glb``) formats.
+- ``.blend`` (Blender). This works by calling Blender to export to glTF in a
+  transparent manner (requires Blender to be installed).
+- DAE (COLLADA), an older format that is fully supported.
+- OBJ (Wavefront) format + their MTL material files. This is also fully
+  supported, but pretty limited given the format's limitations (no support for
+  pivots, skeletons, animations, UV2, PBR materials, ...).
+- FBX, supported via `FBX2glTF <https://github.com/godotengine/FBX2glTF>`__ integration.
+  This requires installing an external program that links against the proprietary FBX SDK,
+  so we recommend using other formats listed above (if suitable for your workflow).
+
+Copy the scene file together with the textures and mesh data (if separate) to
+the project repository, then Godot will do a full import when focusing the
+editor window.
+
+Exporting glTF 2.0 files from Blender (recommended)
+---------------------------------------------------
+
+There are 3 ways to export glTF files from Blender:
+
+- As a glTF binary file (``.glb``).
+- As a glTF text-based file with embedded binary data (``.gltf`` file)
+- As a glTF text-based file with separate binary data and textures (``.gltf``
+  file + ``.bin`` file + textures).
+
+glTF binary files (``.glb``) are the smallest of the three options. They include
+the mesh and textures set up in Blender. When brought into Godot the textures
+are part of the object's material file.
+
+glTF embedded files (``.gltf``) function the same way as binary files. They
+don't provide extra functionality in Godot, and shouldn't be used since they
+have a larger file size.
+
+There are two reasons to use glTF with the textures separate. One is to have the
+scene description in a text based format and the binary data in a separate
+binary file. This can be useful for version control if you want to review
+changes in a text-based format. The second is you need the texture files
+separate from the material file. If you don't need either of those, glTF binary
+files are fine.
```

Comment:
We shouldn't be recommending gltf binary files. I think the recommendation is gltf separate, because the texture extraction doesn't have to be done by the engine.

It's not the end of the world, just something I noticed.